packages/grove-water-level-sensor/rollup.config.js

```diff
@@ -0,0 +1,10 @@
+import { module as input, main as outputFile } from "./package.json";
+
+export default {
+  input,
+  output: {
+    file: outputFile,
+    format: "umd",
+    name: "grove-water-level-sensor"
```

Comment:
ここはUMDのグローバル変数名を指定するのですが、おそらくハイフンを含めると構文エラーになる恐れがあります

なので以下のように変更提案します:

```suggestion
    name: "GroveWaterLevelSensor"
```

使い方 (UMD の場合):

```js
const gwls = new GroveWaterLevelSensor(...);
```

参考リンク: https://rollupjs.org/configuration-options/#output-name